docs/key_concepts.rst

```diff
@@ -82,6 +82,8 @@ Each worker executes in its own greenthread. The maximum number of concurrent wo
 
 Workers are stateless so are inherently thread safe, but dependencies should ensure they are unique per worker or otherwise safe to be accessed concurrently by multiple workers.
 
+Unfortunately, many C-extensions that are using sockets and that would normally be considered thread-safe may not work with greenthreads. Among them `librabbitmq <https://pypi.python.org/pypi/librabbitmq>`_, `MySQLdb <http://mysql-python.sourceforge.net/MySQLdb.html>` and others.
```

Comment:
Please change

"Unfortunately, many C-extensions" => "Note that many C-extensions"
"Among them" => "Among them are"

And then add

"Modules with C-extensions that are not "green" may be wrapped in an eventlet `thread-pool <http://eventlet.net/doc/threading.html>`."